api/v1alpha1/sku_handler/cloud_sku_handler.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package skuhandler
+
+type CloudSKUHandler interface {
+	GetSupportedSKUs() string
```

Comment:
Why return a string, instead of an array of strings?

Reply:
This method is currently used to print out the supported sku list to users: https://github.com/Azure/kaito/blob/6be8a0d57225f776d1fe01bd6069f7d3f894eff0/api/v1alpha1/workspace_validation.go#L274

Reply:
Are you going to use a string type to represent a list? This is a poor design for an interface method.

Reply:
That makes sense, I have changed it to an array